smart-contracts/.openzeppelin/polygon-mumbai.json

```diff
@@ -14,6 +14,21 @@
       "address": "0x8c5D54B2CAA4C2D08B0DDF82a1e6D2641779B8EC",
       "txHash": "0x08b3b4dee1de1508132e6bb5ca2b5027dd8e737bed797f69bf9639a73f43bdbf",
       "kind": "transparent"
+    },
```

Comment:
Do we want to keep these changes? I think you just redeployed for tests? Probably not worth to keep track?

Reply:
yep, removed. OZ renamed the file automatically when deploying